src/HotChocolate/Core/src/Types.Scalars/LocalDateType.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
+using System.Globalization;
+using HotChocolate.Language;
+
+namespace HotChocolate.Types.Scalars
+{
+    /// <summary>
+    /// The `LocalDate` scalar type represents a ISO date string, represented as UTF-8
+    /// character sequences YYYY-MM-DD. The scalar follows the specification defined in
+    /// <a href="https://tools.ietf.org/html/rfc3339">RFC3339</a>
+    /// </summary>
+    public class LocalDateType  : ScalarType<DateTime, StringValueNode>
+    {
+        private const string _localFormat = "yyyy-MM-dd";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalDateType"/> class.
+        /// </summary>
+        public LocalDateType()
+            : this(
+                WellKnownScalarTypes.LocalDate,
+                ScalarResources.LocalDateType_Description,
+                BindingBehavior.Implicit)
```

Comment:
We need to register this type by default as explicit
```suggestion
                ScalarResources.LocalDateType_Description)
```